tpchgen-cli/src/main.rs

```diff
@@ -90,13 +90,15 @@ struct Cli {
     #[arg(short = 'T', long = "tables", value_delimiter = ',', value_parser = TableValueParser)]
     tables: Option<Vec<Table>>,
 
-    /// Number of parts to generate (manual parallel generation)
-    #[arg(short, long, default_value_t = -1)]
-    parts: i32,
+    /// Number of partitions to generate (manual parallel generation)
+    #[arg(short, long)]
+    parts: Option<i32>,
```

Comment:
the main purpose of this PR is to make these options. The rest is just error checking and tests